src/vmemcache_repl.c

```diff
@@ -200,6 +228,16 @@ repl_p_lru_insert(struct repl_p_head *head, void *element)
 static int
 repl_p_lru_use(struct repl_p_head *head, struct repl_p_entry *entry)
 {
+	if (entry == NULL)
```

Comment:
`ASSERTne(entry, NULL);`
You could also add:
`ASSERT(entry exists in head);`
And then make this function return void.